### DISCUSSION TITLE
ability for negative cost

### DISCUSSION THREAD

**Discussion by 19Andy89** [hargata/lubelog#621] [Ideas]
I sold my car and now i got refunded taxes and insurance. is there an easy way to input these refunds? easiest way would be to accept negative cost-values.

Is this possible to implement?

**Comment by Forceu** [comment]
A fix for this was proposed in #843

**Comment by jsddsfoh** [comment]
I sold the set alloy wheels that came with the car, so there are some ways to "earn" money with a car xD (in reality of course I bought different styled alloy wheels...)
Edit: It's a used car bought 6,5 years ago, not new ordered.
